lib/src/Base/Graph/Cloud.cxx

```diff
@@ -112,6 +112,22 @@ String Cloud::__repr__() const
   return oss;
 }
 
+/* Accessor for boundingbox */
+Cloud::BoundingBox Cloud::getBoundingBox() const
+{
+  BoundingBox boundingBox(BoundingBoxSize);
+  const Point max(data_.getMax());
+  const Point min(data_.getMin());
+  const Scalar x_margin = ResourceMap::GetAsScalar("Cloud-HorizontalMargin");
+  const Scalar y_margin = ResourceMap::GetAsScalar("Cloud-VerticalMargin");
+  const Point delta(max - min);
+  boundingBox[0] = min[0] - x_margin * delta[0];
+  boundingBox[1] = max[0] + x_margin * delta[0];
+  boundingBox[2] = min[1] - y_margin * delta[1];
+  boundingBox[3] = max[1] + y_margin * delta[1];
+  return boundingBox;
```

Comment:
I dont know if it is the best way; what is the issue if delta is biggest and there is only a few points ?